services/github-service-api/src/main/java/org/kontinuity/catapult/service/github/api/GitHubRepository.java

```diff
@@ -1,7 +1,12 @@
+/**
```

Comment:
We're not EPL. :)  And Legal advises that we don't need / shouldn't have copyright notices on source code files.